### PR TITLE
Added ability to support multiple help center links

### PR DESCRIPTION
**Pull request by theghall**

### CODE REVIEW

File: addon/components/help-widget/help.js [hunk 1]
```diff
@@ -13,7 +13,8 @@ export default class HelpWidgetHelpComponent extends Component {
   @tracked suggestions = [];
 
   get helpCenterLink() {
-    const url = this.intl.t('ember-help-widget.help-center-url');
+    const helpUrlkey = this.args?.helpCenterUrlKey || 'help-center-url';
+    const url = this.intl.t('ember-help-widget.'.concat(helpUrlkey));
```
Comment (kobsy) [inline addon/components/help-widget/help.js:17]: ⬇️ ⬇️ You can actually use ES6 backticks string interpolation here, and it's a bit more readable:
```
this.intl.t(`ember-help-widget.${helpUrlkey}`);
```

**Review by kobsy — APPROVED** [review]
🆒 Good call 👍

**Comment by kobsy** [comment]
Looks like this is what's failing the test on `ember-release` 😬 
> To use these addons, your app needs ember-auto-import >= 2: ember-source

You _may_ be able to just add an entry for `ember-auto-import` in the Ember try config, but this is a jump we're going to have to look at making sooner or later... 😞

**Comment by kobsy** [comment]
From looking at the `ember-auto-import` [upgrade guide](https://github.com/ef4/ember-auto-import/blob/main/docs/upgrade-guide-2.0.md), we should probably release a new version that uses v2, since that's essentially a breaking change for consuming apps (can't coexist with v1.x in the host app):

> addons that upgrade to ember-auto-import >= 2 will only work in apps that have ember-auto-import >= 2, so they should do their own semver major releases when they upgrade.

@theghall, if you think this is ready otherwise, I can go ahead and merge and cut a new version. We'll just have to plan on bumping `ember-auto-import` and the version number when upgrading to the latest version of Ember in our apps.

**Comment by theghall** [comment]
> From looking at the `ember-auto-import` [upgrade guide](https://github.com/ef4/ember-auto-import/blob/main/docs/upgrade-guide-2.0.md), we should probably release a new version that uses v2, since that's essentially a breaking change for consuming apps (can't coexist with v1.x in the host app):
> 
> > addons that upgrade to ember-auto-import >= 2 will only work in apps that have ember-auto-import >= 2, so they should do their own semver major releases when they upgrade.
> 
> @theghall, if you think this is ready otherwise, I can go ahead and merge and cut a new version. We'll just have to plan on bumping `ember-auto-import` and the version number when upgrading to the latest version of Ember in our apps.

And after I _thought_ I had made sure that I made a non-breaking change....

**Comment by kobsy** [comment]
Looking at it more closely, it looks like `ember-release` is now pointing at v4.0.0. So it's not you that changed, it's Ember. 😆

**Pull request merged**